Signal/src/AppDelegate.m

```diff
@@ -239,7 +239,11 @@ - (void)applicationDidBecomeActive:(UIApplication *)application {
                                                [TSSocketManager becomeActiveFromForeground];
                                                [[Environment getCurrent].contactsManager verifyABPermission];
                                            }];
-
+    
+    if ([TSAccountManager isRegistered]) {
+        [[PushManager sharedManager] applicationDidBecomeActive];
```

Comment:
It might make sense to move the messageFetcherJob to the Environment, and have this code and the push manager access the fetcher job from the Environment.

I'm also OK with it here given the cost/benefit of hammering things onto the environment.